readme.md

```diff
@@ -44,6 +52,43 @@ Helpful tools for students with engineering background
 - [Wolfram Mathematica](https://www.wolfram.com/mathematica/pricing/students-individuals.php) — tools for technical computing. *Not completly free, but with high discount.*
 
 
+## Education
+
+Platforms for education
+
+- [Invision](https://www.invisionapp.com/education-signup) — the digital product design platform powering the world’s best user experiences
+
+- [AWS Educate](https://aws.amazon.com/education/awseducate/) — resources needed to accelerate cloud-related learning.
+
+
+## Analysis and Finance
+- [Tableau](https://www.tableau.com/academic/students) — Prepare data for analysis, powerful drag-and-drop analytics, share data and insights.
+
+- [YNAB Student Program ](https://www.youneedabudget.com/college/) — personal finance software
```

Comment:
Is it useful to unite analysis and finance tools to one section? Maybe it is better to have more general section